DeAceroDLabs/marketplace-frontend
# Feature/dmp 552 Product description on Product Detail view.

## What is done

[[DMP-552] Product description on Product Detail view.](https://deacero.atlassian.net/browse/DMP-552)

## How is done

- Added product description on `ProductDetail` component view.

## How to test

1. Run the project via `npm start`.
2. Click on any product.
3. You should see product description.
